### DISCUSSION TITLE
One footnote for two different types of locations

### DISCUSSION THREAD

**Discussion by rjazwiec** [rstudio/gt#1775] [Q&A] [answered]
Is there a way to make one footnote with two different reference points?
I've tried to do it like that:
```locations = c(cells_body(columns = reason, rows = 5), cells_column_spanners(spanners = "tot"))```
but it doesn't work

I know I can do it manually leaving only `cells_body(columns = reason, rows = 5)` and adding superscript 1 to the tab spanner label but I don't like that solution.

Is there a better method to make footnote with many reference points in different parts of the table?

**Comment by rich-iannone** [comment, marked as answer]
You are pretty close! You just need to use `list()` instead of `c()` for your `locations` calls.

This works for what you're trying to do:

```r
library(gt)

exibble |>
  gt() |>
  tab_footnote(
    footnote = "My footnote",
    locations = list(
      cells_body(columns = num, rows = 1),
      cells_column_labels(columns = fctr)
    )
  )
```

<img width="809" alt="image" src="https://github.com/user-attachments/assets/4467aa48-f834-4c4b-8b18-6ca08c2ec332">

**Reply by rjazwiec** [reply]
Thank you.
